### PR TITLE
Add support for Ukrainian

### PR DESCRIPTION
**Pull request by nvolk**
Support Ukrainian.

NB 1! Our projects seems to use Pasi's personal version. This may not fo automagically to npm.

NB 2! I didn't update dev dependencies. They are very old and complain a lot. However, as they are dev dependencies, we can live with these.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,34 +1,139 @@
...
+  ],
+  "homepage": "https://github.com/NatLibFi/sfs-4900",
+  "bugs": {
+    "url": "https://github.com/NatLibFi/sfs-4900/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:natlibfi/sfs-4900.git"
+  },
   "main": "lib/transliteration.js",
```
Comment (natlibfi-jonollil) [inline package.json:29]: "main": "dist/transliteration.js",

File: package.json [hunk 2]
```diff
@@ -1,34 +1,139 @@
...
+    "cover:unit": "nyc --silent npm run test:base",
+    "cover:report": "nyc report"
+  },
+  "originalScripts": {
     "prepublish": "babel src --out-dir lib",
     "test": "npm run lint && npm run test:unit",
     "test:unit": "BABEL_ENV=test mocha --compilers js:babel-core/register test/**.spec.js",
     "test:unit:watch": "npm run test:unit -- --watch",
     "lint": "eslint ."
   },
```
Comment (natlibfi-jonollil) [inline package.json:56]: Can be deleted

File: package.json [hunk 3]
```diff
@@ -1,34 +1,139 @@
...
+    "@babel/cli": "^7.25.7",
+    "@bebel/core": "^7.25.7",
+    "eslint": "^8.57.1",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-stage-2": "^6.18.0",
     "chai": "^3.5.0",
-    "eslint": "^3.9.0",
-    "mocha": "^3.1.2"
+    "mocha": "^10.7.3"
+  },
```
Comment (natlibfi-jonollil) [inline package.json:65]: can be deleted

File: package.json [hunk 4]
```diff
@@ -1,34 +1,139 @@
...
+    "babel-plugin-istanbul": "^7.0.0",
+    "babel-plugin-rewire": "^1.2.0",
+    "chai": "^4.5.0",
+    "chai-as-promised": "^7.1.2",
+    "cross-env": "^7.0.3",
+    "eslint": "^8.57.1",
+    "fetch-mock": "^11.1.5",
+    "mocha": "^10.7.3",
+    "nyc": "^17.1.0"
+  },
```
Comment (natlibfi-jonollil) [inline package.json:82]: "@babel/cli": "^7.25.7",
    "@babel/core": "^7.25.8",
    "@babel/preset-env": "^7.25.8",
    "@natlibfi/eslint-config-melinda-backend": "^3.0.5",
    "babel-plugin-istanbul": "^7.0.0",
    "chai": "^4.5.0",
    "cross-env": "^7.0.3",
    "eslint": "^8.57.1",
    "mocha": "^10.7.3",
    "nyc": "^17.1.0"

File: package.json [hunk 5]
```diff
@@ -1,34 +1,139 @@
...
+        "@babel/preset-env",
+        {
+          "targets": "maintained node versions"
+        }
+      ]
+    ],
+    "env": {
+      "test": {
+        "plugins": [
+          "rewire",
```
Comment (natlibfi-jonollil) [inline package.json:100]: Remove

File: package.json [hunk 6]
```diff
@@ -1,34 +1,139 @@
...
+    ],
+    "inline-diffs": true,
+    "maxDiffSize": 25000,
+    "bail": true,
+    "exit": true
+  },
+  "nodemonConfig": {
+    "exec": "npm run test:dev",
+    "watch": [
+      "lib/*",
```
Comment (natlibfi-jonollil) [inline package.json:121]: `"src/*",`

**Review by natlibfi-jonollil — CHANGES_REQUESTED** [review]
Add .github folder and contents like this https://github.com/NatLibFi/melinda-record-import-commons-js

File: package.json [hunk 7]
```diff
@@ -1,34 +1,139 @@
 {
-  "name": "sfs4900",
-  "version": "1.0.0",
-  "description": "Transliteration from russian text to finnish as defined in SFS-4900 standard",
+  "name": "@natlibfi/sfs-4900",
+  "version": "1.0.1",
+  "license": "MIT",
+  "description": "Transliteration from cyrillic (Russian, Ukrainian) text to latin (Finnish) as defined in the SFS-4900 standard",
+  "author": "Pasi Tuominen <pasi.tuominen@gmail.com>",
```
[the remark this answers is not in the document]
Comment (nvolk) [reply package.json]: ok

File: package.json (hunk 2 above)
Comment (nvolk) [reply package.json]: deleted

File: package.json (hunk 3 above)
Comment (nvolk) [reply package.json]: obsolete duplicate lines removed

File: package.json (hunk 4 above)
Comment (nvolk) [reply package.json]: unneeded deps removed

File: package.json (hunk 6 above)
Comment (nvolk) [reply package.json]: changed

File: package.json (hunk 5 above)
Comment (nvolk) [reply package.json]: removed

File: package.json (hunk 1 above)
Comment (nvolk) [reply package.json]: done

**Pull request merged**